services/cd-service/pkg/repository/transformer_test.go

```diff
@@ -1393,6 +1393,212 @@ func TestCreateApplicationVersionCommitPath(t *testing.T) {
 	}
 }
 
+type ContentRequirement struct {
```

Comment:
nitpick: maybe rename to "FileContent", or "FileWithContent", or "ExpectedFileContent"?